debian/pat-configure.1

```diff
@@ -0,0 +1,17 @@
+.TH PAT 1 "2017-09-04" "Linux" "Pat Configure"
+.SH NAME
+pat configure \- opens pats configuration file using nano
```

Comment:
> using nano

Although this is true on many Linux systems, it depends on your system configuration and environment (https://github.com/la5nta/pat/blob/master/main.go#L488).

The default fallback is in fact `vi`, but I would prefer something like "opens Pats configuration file using the system default editor".

Reply:
Okay, I'll change that.